@package magento/framework
===================================================================
--- Encryption/Encryptor.php
+++ Encryption/Encryptor.php
@@ -149,10 +149,11 @@
         $this->keyVersion = count($this->keys) - 1;
         $this->keyValidator = $keyValidator ?: ObjectManager::getInstance()->get(KeyValidator::class);
         $latestHashVersion = $this->getLatestHashVersion();
+        /*
         if ($latestHashVersion === self::HASH_VERSION_ARGON2ID13) {
             $this->hashVersionMap[self::HASH_VERSION_ARGON2ID13] = SODIUM_CRYPTO_PWHASH_ALG_ARGON2ID13;
             $this->passwordHashMap[self::PASSWORD_VERSION] = self::HASH_VERSION_ARGON2ID13;
-        }
+        }*/
     }
 
     /**
@@ -162,9 +163,10 @@
      */
     public function getLatestHashVersion(): int
     {
+        /*
         if (extension_loaded('sodium')) {
             return self::HASH_VERSION_ARGON2ID13;
-        }
+        }*/
 
         return self::HASH_VERSION_SHA256;
     }
